crosswalks/MODS.csv

```diff
@@ -0,0 +1,68 @@
+﻿Property,MODS
+codeRepository,relatedItem/[@type='Host']
+programmingLanguage,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,location/url[@displayName='Download URL']
+fileSize,physicalDescription/extent
+installUrl,location/url[@displayName='Install URL']
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,description
+softwareHelp,relatedItem
+softwareRequirements,
+softwareVersion,originInfo/edition
+storageRequirements,
+supportingData,relatedItem[@type='references']
+author,name/role/roleTerm[@type='code']/aut
+citation,relatedItem[@otherType='references']
+contributor,name/role/roleTerm[@type='code']/ctb
+copyrightHolder,accessCondition/rights.holder/name
+copyrightYear,accessCondition/creation/year.creation
+dateCreated,originInfo/dateCreated
+dateModified,originInfo/dateModified
+datePublished,originInfo/dateIssued
+editor,name/role/roleTerm[@type='code']/edt
+encoding,
+fileFormat,typeOfResource
+funder,name/role/roleTerm[@type='code']/fnd
+keywords,subject
+license,accessCondition[@type='use and reproduction']
+producer,name/role/roleTerm[@type='code']/pro
+provider,name/role/roleTerm[@type='code']/prv
+publisher,originInfo/publisher
+sponsor,name/role/roleTerm[@type='code'
+version,originInfo/edition
+isAccessibleForFree,accessCondition[@type='accessible for free']/yes
+isPartOf,part
+hasPart,part

```

Comment:
```suggestion
isPartOf,part
hasPart,
```

MODS's `<part>` can't mean both, can it? According to [this](https://www.loc.gov/standards/mods/userguide/part.html), it seems to be the former.

Reply:
```suggestion
isPartOf,part
hasPart,relatedItem[@type='constituent']
```
I agree, do you think the constituent type for the [related item](https://www.loc.gov/standards/mods/userguide/relateditem.html) tag be an appropriate mapping?